static/styles/left_sidebar.css

```diff
@@ -199,33 +198,34 @@ li.show-more-topics {
 
 #private_messages_section {
     margin-bottom: $sections_vertical_gutter;
+    margin-right: 12px;
 
-    .fa-caret-down,
-    .fa-chevron-left {
+    #toggle_private_messages_section_icon {
         opacity: 0.7;
+        margin-left: 3px;
+        min-width: 11px;
```

Comment:
We would not want the `PRIVATE MESSAGES` heading to shift right-left on toggling PM section, hence assigning a `min-width` to it.